src/tests/ftest/recovery/check_policy.py

```diff
@@ -0,0 +1,165 @@
+"""
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from exception_utils import CommandFailure
+from general_utils import report_errors
+from recovery_test_base import RecoveryTestBase
+
+
+class DMGCheckPolicyTest(RecoveryTestBase):
```

Comment:
In https://github.com/daos-stack/daos/pull/16656 this is replaced by:
```suggestion
class DMGCheckPolicyTest(TestWithServers):
```